features/view-site-in-lang-i18n-l10n.feature

```diff
@@ -0,0 +1,35 @@
+Feature: As a visitor
+  In order to view the site in my language
+  I need to be able choose either Swedish or English for the site
+
+  PT: https://www.pivotaltracker.com/story/show/133316647
+
+  Background:
+    Given I am Logged out
+
+
+  Scenario: Default language is Swedish
+    Given I am on the "all companies" page
+    Then I should see "Hitta H-märkt företag"
+    And I should not see "Swedish flag"
+    And I should see "English flag"
+    And I should see t("theme_copyright", locale: :sv)
+
+
+  Scenario: Visitor switches the site language from English to Swedish
+    Given I am on the "all companies" page
+    When I click on "change-lang-to-english"
+    When I click on "change-lang-to-svenska"
```

Comment:
Yup, that's exactly why.  But @patmbolger you're point is a good one. 
I'll write a step so that the line in the scenario is readable (uses Swedish flag) and put the implementation details into the step.  
There will be more PRs for this so I'll do it in those.